src/components/CreateListingPage/CreateListing.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledInput = styled.input`
+  height: 32px;
+  border: 2px solid #abc38e;
+  border-radius: 11px;
+  margin: 5px 0;
+  padding: 5px;
+`;
+
+const StyledInputButtonGroup = styled.div``;
+
+const StyledLabel = styled.label`
+  padding: 5px;
+`;
+
+const StyledInputButton = styled.input`
+  border: 2px solid #ffd084;
+  border-radius: 11px;
+  padding: 8px;
+  margin: 8px;
+  background-color: ${props => (props.filling ? '#ffd084' : 'transparent')};
+  font-family: 'PT Mono', monospace;
+`;
+
+const StyledButton = styled.button`
+  background-color: #abc38e;
+  border-radius: 11px;
+  padding: 9px;
+  font-size: 18px;
+  font-family: 'PT Mono', monospace;
+  margin-top: 15px;
```

Comment:
Tip: Sort your css-props alphabetical or with an other system to have a better overview

Reply:
Done